ci/raydepsets/configs/rayllm.depsets.yaml

```diff
@@ -18,7 +18,6 @@ build_arg_sets:
     - --extra-index-url https://download.pytorch.org/whl/${CUDA_CODE}
   build_arg_sets:
     - cpu
-    - cu121
```

Comment:
I don't think we need this cuda version anymore since the torch 2.8.0 doesn't support it anyways?